xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1349,6 +1349,67 @@ class FlashAttentionBMMScalePaddingMaskSoftmaxBMMF8
   }
 };
 
+class FlashAttentionBMMScaleSoftmaxDropoutBMM
+    : public MultiHeadedAttentionTest {
+ protected:
+  const std::string  // NOLINT
+  GetModuleFlash_Attention_Training_BMM1_Softmax_Dropout_BMM2_HloString_BF16() {  // NOLINT
+    const std::string hlo_text = R"(
```

Comment:
Why the NOLINT and why the `const std::string`? This has no real benefit here?

I recommend making this a constant instead of a function:

`constexpr absl::string_view kModuleFlashAttentionTrainingBMM1SoftmaxDropoutBMM2HloStringBF16 = R"(...)";`

Reply:
Fixed.